libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -496,6 +474,7 @@ - (void)logoutUser:(SFUserAccount *)user
             [delegate authManagerDidLogout:weakSelf];
         }
     }];
+    user.userLoggingOut = NO;
```

Comment:
And as an addendum, we'd also add this line just before [Line 451](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1939/files#diff-4aedc82766c844e15a84507c44206a64R451).